### ISSUE TITLE
[bug] subsequent runs fail validation

### ISSUE THREAD

**Issue by NickLarsenNZ** [StephaneBour/actions-calver#4]
I have configured my workflow like so:

```yml
    - name: Calver Release
      uses: StephaneBour/actions-calver@master
      id: calver
      with:
        date_format: "stage-%Y-%m-%d"
        release: ${{ github.event_name == 'push' && github.ref == 'refs/heads/stage' }} # disable, or update to stage?
        release_branch: stage
      env:
        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

- When I do an initial run, I get a tag `stage-2021-12-16`.
- I run again (even if there is no new commit) and get tag `stage-2021-12-16.1`.
- On another run (in this case via a new commit), the workflow incorrectly completes successfully, and an error is shown:
   ```
   Last hash : d5ce81bd712de5c65c05d029e194b75d8d2ba8d4
   Last release : stage-2021-12-16
   Last major release : stage-2021-12-16
   Minor release
   Next release : stage-2021-12-16.1
   ##  (2021-12-16)

   * noop ([e8067be](https://github.com/me/playground/commit/e8067be))
   * remove %V (posix week) ([d5ce81b](https://github.com/me/playground/commit/d5ce81b))
   Create release : true
   { "tag_name": "stage-2021-12-16.1", "target_commitish": "stage", "name": "stage-2021-12-16.1", "body": "## (2021-12-16)\n\n* noop ([e8067be](https://github.com/me/playground/commit/e8067be))\n* remove %V (posix week) ([d5ce81b](https://github.com/me/playground/commit/d5ce81b))", "draft": false, "prerelease": false }
   {
     "message": "Validation Failed",
     "errors": [
       {
         "resource": "Release",
         "code": "already_exists",
         "field": "tag_name"
       }
     ],
     "documentation_url": "https://docs.github.com/rest/reference/repos#create-a-release"
   }
   ```

_FYI, `noop` was the commit message_

**Comment by NickLarsenNZ** [comment]
I've tried with `release: false`, but then no tag is pushed.

**Comment by stevethomas** [comment]
https://github.com/StephaneBour/actions-calver/blob/master/entrypoint.sh#L32

It seems `MINOR_LAST_RELEASE` only works with YYYY? 

In my output i'm getting:

```
Last release : 
Last hash : 
Last major release : 
Next release : 22.26
```
Even though the first release was 22.26.

For now i've just used `date_format: "%y.%W.%H%M"` to get around the last release parsing. 

Not sure on the calver conventions, but perhaps you could always append a minor version starting at 1?

**Comment by KrisM4c** [comment]
@NickLarsenNZ its due to the regex parsing - https://github.com/StephaneBour/actions-calver/blob/master/entrypoint.sh#L32

Changing the `-`'s to `.`'s fixed my issue. Also the string needs to start with `20`

You can see the regex matching here:
WORKING:
<img width="188" alt="Screen Shot 2022-09-30 at 10 21 21" src="https://user-images.githubusercontent.com/34947079/193238423-991c5805-cde0-4717-97cb-6bce38301cad.png">


NOT WORKING:
<img width="229" alt="Screen Shot 2022-09-30 at 10 20 45" src="https://user-images.githubusercontent.com/34947079/193238294-2e6e8e00-722e-4e14-b497-97a62b208b1b.png">

**Comment by miigotu** [comment]
Fixed in #16
